src/language/languageServiceHost.ts

```diff
@@ -17,18 +17,16 @@
 import * as Lint from "../lint";
 import * as ts from "typescript";
 
-export function createLanguageServiceHost(fileName: string, source: string) {
-    const host: ts.LanguageServiceHost = {
+export function createLanguageServiceHost(fileName: string, source: string): ts.LanguageServiceHost {
+    return {
         getCompilationSettings: () => Lint.createCompilerOptions(),
         getCurrentDirectory: () => "",
         getDefaultLibFileName: () => "lib.d.ts",
         getScriptFileNames: () => [fileName],
-        getScriptSnapshot: (name) => ts.ScriptSnapshot.fromString(name === fileName ? source : ""),
+        getScriptSnapshot: (name: string) => ts.ScriptSnapshot.fromString(name === fileName ? source : ""),
         getScriptVersion: () => "1",
-        log: (message) => { /* */ }
+        log: () => { /* */ }
```

Comment:
Feel free to in a follow-up if you want, definitely not a blocking thing for me though. Going to go ahead and merge this